api/utils/reflect.go

```diff
@@ -0,0 +1,244 @@
+package utils
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+type DbTag struct {
+	Constraint string
+}
+
+func GetFieldTags(t reflect.Type) (fieldToJsonTags map[string]string,
```

Comment:
This file has very long functions. I can't understand each one of them but see if you could divide them into logically useful functions